Spigot-Server-Patches/0334-Configurable-speed-for-water-flowing-over-lava.patch

```diff
@@ -0,0 +1,65 @@
+From 35c565d267a562633607766e1264ac0cd4c01d87 Mon Sep 17 00:00:00 2001
+From: Colin Godsey <crgodsey@gmail.com>
```

Comment:
Also, we need to keep the original author / timestamp of this patch.

https://github.com/PaperMC/Paper/blob/pre/1.13/removed/server/0017-Configurable-speed-for-water-flowing-over-lava.patch